app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+
+    if command_parts[0] != "mv" or len(command_parts) != 3:
+        raise ValueError("Incorrect command input.")
+
+    old_file, new_file = command_parts[1:]
+    directory_parts = new_file.split("/")[:-1]
+
+    current_file_path = ""
+
+    for directory in directory_parts:
+        current_file_path = os.path.join(current_file_path, directory)
+        os.makedirs(current_file_path, exist_ok=True)
```

Comment:
You don't need `for` loop here. Read how `os.makedirs()` works

Reply:
Ok, I rewrote it using the condition